cli/reference/cmdline/run.md

```diff
@@ -298,6 +298,40 @@ terramate run [options] -- <cmd ...>
 
   Use Terragrunt to generate the Terraform/OpenTofu Plan file.
 
+### TMC: Deployment Targets
+
+**Note:** This is an upcoming experimental feature that is subject to change in the future. To use it now, you must enable the project config option `terramate.config.experiments = ["targets"]`
+
+Terramate supports synchronizing stacks to different deployment targets, each having an isolated state.
+This enables deploying to multiple environments, i.e. `testing`, `staging`, `prod`, or to multiple regions.
+
+Before using this feature, you must enable targets for your project by setting the config option `terramate.config.cloud.targets.enabled = true`.
+Once enabled, the following options are available:
+
+- `--target <target-identifier>`
+
+  Synchronize data to the Terramate Cloud (TMC) for the given deployment target. Stacks for each target are isolated from each other and have a separate status.
+
+  Once the targets feature is enabled, setting a target is _required_ for any option that uses Terramate Cloud, to safeguard against accidently synchronizing to an implicit default target.
+
+  Target identifiers must match the pattern `[a-zA-Z0-9_-]{1,64}`.
+
+  This flag is supported in combination with `--sync-deployment`, `--sync-drift-status`, `--sync-preview`, and `--status`.
+
+- `--from-target <target-identifier>`
+
+  Attempt to migrate stacks from `--from-target <old-target>` to `--target <new-target>`.
+  
+  The rules for this migration are applied per stack and are defined as follows:
+  - If the stack already exists in `new-target`, the migration is skipped.
+  - If the stack doesn't exist in `old-target`, the migration is skipped.
+  - If used together with `--sync-preview`, the migration is skipped.
+  - Otherwise, the stack is moved/renamed from `old-target` to `new-target`.
```

Comment:
```suggestion
  The rules for this migration are applied per stack as follows:
- Skip the migration if the stack already exists in `new-target`.
- Skip the migration if the stack doesn't exist in `old-target`.
- Skip the migration if used with `--sync-preview`.
- Otherwise, move or rename the stack from `old-target` to `new-target`.
```

Reply:
Left the moved/renamed, its meant as a synonym rather than alternatives.